crates/db_views/src/post_report_view.rs

```diff
@@ -19,6 +38,9 @@ pub struct PostReportView {
   pub community: CommunitySafe,
   pub creator: PersonSafe,
   pub post_creator: PersonSafeAlias1,
+  pub creator_banned_from_community: bool,
+  pub my_vote: Option<i16>,
+  pub counts: PostAggregates,
```

Comment:
What are these fields for?

Reply:
Necessary for the front end, otherwise it wouldn't be able to build the entire post previews from the report page (it'd be missing scores, your vote, etc)